src/components/radio/radio.ts

```diff
@@ -29,8 +29,8 @@ export default class SlRadio extends LitElement {
 
   @query('input[type="radio"]') input: HTMLInputElement;
 
-  // @ts-ignore
-  private formSubmitController = new FormSubmitController(this, {
+  // @ts-expect-error -- Controller is currently unused
```

Comment:
Honestly, this seems like a misuse of a constructor for its side effects. This should probably be a method call in the constructor instead. See [no-new](https://eslint.org/docs/rules/no-new). I wasn't sure what to do here and this was a last minute rebase change so I didn't spend much time on it.

Reply:
It's not really a misuse, but more to keep things consistent with how other controllers are instantiated. It's possible we'll need to reference the controller at some point, hence the consistent assignments even when unused. Most Reactive Controllers have methods (e.g. `LocalizeController`), but in this case the defaults are smart enough to not need to access it _today_.

I'm fine with leaving the `@ts-expect-error`. I was just lamenting that it would be nice to have maybe a symbol or something to more elegantly tell TS that this is intentional. 🙃